setup.py

```diff
@@ -134,7 +142,7 @@ def run(self):
               'Topic :: Scientific/Engineering :: Information Analysis',
               'Topic :: Software Development :: Quality Assurance',
               'Topic :: Text Processing :: Linguistic'],
-          cmdclass={'build_manpage': BuildManPage,
+          cmdclass={'docs': BuildDocsCommand,
```

Comment:
bears has docs and test, and now coala has docs and other stuff and test.  That was the easy way to avoid too much differences in the output.  But I'll look into alpha sorting.  I suspect it means moving all these commands into metadata instead of template blocks, and I dont know it is worth the complexity.  But worth looking at again.

Reply:
Maybe we can keep the docs and test in meta, and let the other stuff stay in the template, since those two are same for both?

Reply:
Turns out this isnt easy.